src/style.css

```diff
@@ -2,0 +1,54 @@
+body {
+  margin: 0;
+}
+
+.container {
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  display: flex;
+  flex-direction: column;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 div:nth-child(1) {
```

Comment:
Avoid repetitive code. The `background-position` and `background-repeat` properties are repeated across multiple selectors (`.stars--1 div:nth-child(1)`, `.stars--2 div:nth-child(-n + 2)`, etc.). Since these properties are the same for all the `.stars__star` elements, you should define them once in the `.stars__star` class.